authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,83 @@
+
+## How to set-up a Partner-Managed Application
+An application is viewed as *Partner-Managed* when it is granted elevated permissions to access your customers' data. Note that partner-managed apps can *only* be configured on Partner tenants.
+
+The initial steps required here follow most of the same steps used to register and configure a multi-tenant application:
+
+1. <a href="https://docs.microsoft.com/en-us/azure/active-directory/active-directory-app-registration" target="_newtab">Register your application</a> in your Partner tenant using the <a href="https://portal.azure.com" target="_newtab">Azure Portal</a>. To function as a partner-managed app, an application must be configured as a <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-devhowto-multi-tenant-overview#update-registration-to-be-multi-tenant" target="_newtab">multi-tenant app</a>.
+
+2. Configure your multi-tenant app, again through the Azure Portal, with the *required permissions* it needs using a least privileged approach.
+
+3. Finally grant your Partner-Managed App those configured permissions for all your customers. You can do this by adding the **servicePrincipal** that represents the app to the AdminAgents group in your Partner tenant, using Azure AD powershell V2. You can download and install Azure AD PowerShell V2 from <a href="https://www.powershellgallery.com/packages/AzureAD" target="_newtab">here</a>.  Follow these steps to find the AdminAgents group, the **servicePrincipal** and add it to the group.
+
+    1. Open a PowerShell session and connect to your partner tenant by entering your admin credentials into the sign-in window.
+    ```PowerShell
+    > Connect-AzureAd
+    ```
+    2. Find the group that represents the AdminAgents. 
+    ```PowerShell
+    > $group = Get-AzureADGroup -Filter "displayName eq 'AdminAgents'"  
+    ```
+    3. Find the service principal that has the same *appId* as your app.
+    ```PowerShell
+    > $sp = Get-AzureADServicePrincipal -Filter "appId eq '{yourAppsAppId}'"
+    ```
+    4. Finally, add the service principal to the AdminAgents group.
+    ```PowerShell
+    > Add-AzureADGroupMember -ObjectId $group.ObjectId -RefObjectId $sp.ObjectId
+    ```
+
+## Token acquisition flows
+Partner-managed apps are identical to regular multi-tenant apps in most respects. As such, the token acquisition flows <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a> are the same.
```

Comment:
Are there 3 flows in the sentence starting with "As such"? Do you mean the "token acquisition flow" to be singular and a peer of the other 2 flows? Or do you mean "token acquisition flows" to include the auth code grant flow and client credential flow?
The punctuation is off in either case. If you mean the former, then please insert a comma after "token acquisition flow" (singular "flow"). If you mean the latter, then it should be:
"As such, the token acquisition flows -- authorization grant flow and service-to-service-client credentials flow -- are the same."
"service-to-service", please add hyphens.